src/Files.App/Actions/Open/OpenPropertiesAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Files.App.Commands;
+using Files.App.Contexts;
+
+namespace Files.App.Actions
+{
+	internal class OpenPropertiesAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label { get; } = "OpenProperties".GetLocalizedResource();
+
+		public string Description => "OpenPropertiesDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph { get; } = new RichGlyph(opacityStyle: "ColorIconProperties");
+
+		public bool IsExecutable =>
+			!context.ShellPage.InstanceViewModel.IsPageTypeSearchResults;
+
+		public HotKey HotKey { get; } = new(Keys.P, KeyModifiers.Ctrl);
+
```

Comment:
This shortcut is already assigned to `Open preview pane`

Reply:
Properties shouldn't have a shortcut by default

Reply:
In Explorer its `Alt+Enter`

Reply:
Oh that's right, we might actually use that already.

Reply:
>In Explorer it's Alt+Enter

How can I specify? I don't know which enums represent them.

Reply:
You should use `new HotKey(Keys.Enter, KeyModifiers.Menu)`